concent_api/api_testing_common.py

```diff
@@ -268,23 +292,24 @@ def create_client_auth_message(client_priv_key: bytes, client_public_key: bytes,
     return dump(client_auth, client_priv_key, concent_public_key)
 
 
-def parse_command_line(command_line: list) -> str:
-    if len(command_line) <= 1:
-        sys.exit('Not enough arguments')
-
-    if len(command_line) >= 3:
-        sys.exit('Too many arguments')
-
-    cluster_url = command_line[1]
-    return cluster_url
-
-
 def parse_arguments() -> Tuple:
     parser = argparse.ArgumentParser()
     parser.add_argument("cluster_url")
     parser.add_argument("tc_patterns", nargs='*')
+    parser.add_argument(
+        '-c1',
+        '--concent_1_version',
```

Comment:
I think it should be named `--concent_1_golem_messages_version`. Same for second one.

Reply:
Done